unstructured/cleaners/extract.py

```diff
@@ -44,3 +52,19 @@ def extract_text_after(text: str, pattern: str, index: int = 0, strip: bool = Tr
     _, end = regex_match.span()
     before_text = text[end:]
     return before_text.lstrip() if strip else before_text
+
+
+def extract_email_address(text:str) -> List[str]:
+    return re.findall(EMAIL_ADDRESS_PATTERN, text.lower())
+
+def extract_ip_address(text: str) -> List[str]:
+    return re.findall(IP_ADDRESS_PATTERN_RE, text)
+
+def extract_ip_address_name(text: str) -> List[str]:
+    return re.findall(IP_ADDRESS_NAME_PATTERN, text)
+
+def extract_mapi_id(text:str) -> List[str]:
+    return(re.findall(MAPI_ID_PATTERN, text))
+
+def extract_datetimetz(text:str) -> List[str]:
```

Comment:
Would it be better to return a list of `datetime.datetime` objects for this one?

Reply:
Yes, true!